/project/6065746/pgc/VARNN-Isaac-New/EconDL/EconDL/DataHelpers/DataLoader.py:11: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  x_d_all['L0_HOUST'] = x_d_all['L0_HOUST'].diff()
Folder at results/15jul_cc_test already exists
Loaded results for repeat 0
Not trained yet
DataLoader: Loaded dataset monthly
Trained already
Traceback (most recent call last):
  File "execution_script.py", line 24, in <module>
    RunObj.train_all()
  File "/project/6065746/pgc/VARNN-Isaac-New/EconDL/EconDL/Run.py", line 165, in train_all
    self.train_experiments()
  File "/project/6065746/pgc/VARNN-Isaac-New/EconDL/EconDL/Run.py", line 100, in train_experiments
    ExperimentObj.train(self.dataset, self.device)
  File "/project/6065746/pgc/VARNN-Isaac-New/EconDL/EconDL/Experiment.py", line 181, in train
    self._compile_unconditional_irf_results()
  File "/project/6065746/pgc/VARNN-Isaac-New/EconDL/EconDL/Experiment.py", line 260, in _compile_unconditional_irf_results
    out_repeat = np.load(f'{self.folder_path}/fcast_params_{self.experiment_id}_repeat_{repeat_id}.npz', allow_pickle=True)
  File "/project/6065746/pgc/VARNN-Isaac-New/ENV/lib/python3.7/site-packages/numpy/lib/npyio.py", line 417, in load
    fid = stack.enter_context(open(os_fspath(file), "rb"))
FileNotFoundError: [Errno 2] No such file or directory: 'results/15jul_cc_test/fcast_params_0_repeat_0.npz'
